javascript/javascript2/week2/index2.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+
+
+
+
+
+
+
```

Comment:
So many blank lines that can be deleted. https://ibb.co/MPXjDVD
```suggestion
```

Reply:
hahaha.....the cute dog seems unhappy. I will be careful the next time